### PR TITLE
[full-ci] Switch default config from account/glauth to builtin libregraph/idm

### PR DESCRIPTION
**Pull request by rhafer**
## Description
This switches the default configs of all required service to point to builtin libregraph/idm LDAP server. Also it disables the glauth and accounts services.

~~This PR includes the changes from https://github.com/owncloud/ocis/pull/3323 (which be should hopefully merge before this)~~

Note: For now we still run the webUI test using the accounts service. Once the owncloud-test-middleware is update to use the GraphAPI for user provisioning those tests can be switched over to libregraph/idm as well.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>settingsUITests</strong>](https://drone.owncloud.com/owncloud/ocis/9984/36/1) failed. Further test are cancelled...

File: .drone.star [hunk 1]
```diff
@@ -1693,10 +1710,9 @@ def ocisServer(storage, accounts_hash_difficulty = 4, volumes = [], depends_on =
             "PROXY_ENABLE_BASIC_AUTH": True,
             "WEB_UI_CONFIG": "/drone/src/tests/config/drone/ocis-config.json",
             "IDP_IDENTIFIER_REGISTRATION_CONF": "/drone/src/tests/config/drone/identifier-registration.yml",
-            "OCIS_LOG_LEVEL": "error",
+            "OCIS_LOG_LEVEL": "debug",
```
Comment (micbar) [inline .drone.star:1713]: Is that really necessary? In the past we had real issues with so many log lines hammering our storage system.

Comment (individual-it) [reply .drone.star:1713]: the problem is also that drone truncated the log lines, so you might have trouble to see the first ones

Comment (phil-davis) [reply .drone.star:1713]: yes, I think that drone truncates pipeline step output at around 64k lines! I don't remember if you see the first or last 64k.

Comment (rhafer) [reply .drone.star]: Yeah that line is a left over from debugging a test failure that I couldn't reproduce locally. Removed it now.

**Review by micbar — APPROVED** [review]
LGTM! Awesome

**Pull request merged**